### DISCUSSION TITLE
Type error: 'Html' cannot be used as a JSX component.

### DISCUSSION THREAD

**Discussion by vasily-mishanin** [resend/react-email#1025] [Q&A]
Hi! 
I use  "@react-email/components": "0.0.6", and "react-email": "1.9.3".
I use it in Next.js 13 project.
The folder "react-email-starter" is on the same level with "app" folder (in the root of project).
All templates inside "react-email-starter/emails/" have type errors like "'Body' cannot be used as a JSX component.". All these components (Html, Body e.c.) are imported from '@react-email/components'. 
Help me please! Is this a bug or I can do something?

**Comment by aychoi** [comment]
Same here -- any answers on this front?

**Comment by umar-ahmed** [comment]
You might want to check that you have the correct version of `react` and `@types/react`. I was recently upgrading react-email and I bumped into this issue because the latest typings (`18.2.45` at the time of writing) were in conflict. I downgraded to `18.0.26` and that seems to have resolved the type errors.

This might indicate that there are incompatibilities with react-email's typings for future versions of React, but I didn't have time to investigate further and figure out who should update things
